### PR TITLE
py-seedir: new package

### PR DESCRIPTION
**Pull request by marcella-santana**
Seedir is a Python package for creating, editing, and reading folder tree diagrams.This package facilitates the folder structure understanding, showing all the tree project, with icons, emojis if it's desirable.

@hyviquel

### CODE REVIEW

File: var/spack/repos/builtin/packages/py-seedir/package.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class PySeedir(PythonPackage):
+    """Seedir is a Python package for creating, editing, and reading folder tree diagrams."""
+
+    homepage = "https://earnestt1234.github.io/seedir/seedir/index.html"
```
Comment (manuelakuhn) [inline var/spack/repos/builtin/packages/py-seedir/package.py:12]:
```suggestion
    homepage = "https://github.com/earnestt1234/seedir"
```

Comment (manuelakuhn) [reply var/spack/repos/builtin/packages/py-seedir/package.py:12]: We try to take the same link as specified as homepage on PyPI.

File: var/spack/repos/builtin/packages/py-seedir/package.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    pypi = "seedir/seedir-0.4.2.tar.gz"
+    git = "https://github.com/earnestt1234/seedir.git"
+
+    version("0.4.2", sha256="a549896f37fb31243cf9c8f44405821d9176b2b0c91804f87b7018ea0a738d2b")
+    version("0.3.1", sha256="bbf6aef0fa51cd7e6ba8bdaff67a0913bc7199cd6302a92676fbae409112db54")
+    version("0.2.0", sha256="49d89cb6074267dc76e7048cc5f2904c5eef8657c206d2086ecdd96ba89fc6d6")
+
+    depends_on("python@3.8:", type=("build", "run"))
+    depends_on("py-natsort", type=("build", "run"))
+    depends_on("py-emoji", type=("build", "run"))
```
Comment (manuelakuhn) [inline var/spack/repos/builtin/packages/py-seedir/package.py:22]: This dependency is not needed for the default installation but is an optional dependency (see [pyproject.toml#L27](https://github.com/earnestt1234/seedir/blob/v0.4.2/setup.py#L27).) If you want to add it, you can do that by using a variant.

**Review by manuelakuhn — COMMENTED** [review]
The build dependency on `py-setuptools` is missing.

**Review by bernhardkaindl — COMMENTED** [review]
@marcella-santana Can you apply the fixes which were found by the review? @hyviquel

**Comment by marcella-santana** [comment]
> @marcella-santana Can you apply the fixes which were found by the review? @hyviquel

Hi @bernhardkaindl, my apologies for being late! I will apply the fixes pointed in review ✅

File: var/spack/repos/builtin/packages/py-seedir/package.py (hunk 1 above)
Comment (marcella-santana) [reply var/spack/repos/builtin/packages/py-seedir/package.py:12]: Hmmm right, I will update here ✅

File: var/spack/repos/builtin/packages/py-seedir/package.py (hunk 2 above)
Comment (marcella-santana) [reply var/spack/repos/builtin/packages/py-seedir/package.py:22]: Hmmm I think I've got it, I will change it too ✅

**Comment by bernhardkaindl** [comment]
@marcella-santana Just a reminder

**Comment by bernhardkaindl** [comment]
@marcella-santana Just another reminder.

Note: In case this needs a C compiler, `develop` now wants `depends_on("c", type="build")` (and so on)

**Comment by alalazo** [comment]
Closing, since there has been no activity in a while. Feel free to reopen if you want to continue work on this.

**Pull request closed without merging**